blog/en/blog/2023/04/07/apisix-unity-group-q&a.md

```diff
@@ -0,0 +1,148 @@
+
+**6. What other challenges were you and your team experiencing before using APISIX?**
+
+Setting up the monitoring properly was time-consuming. I am referring to metrics such as SLOs and monitoring of individual endpoints.
+
+## About the Technical Selection
+
+**1. How did you hear about APISIX?**
+
+We have been investing heavily in Kubernetes, and the APISIX was mentioned as one of the projects implementing the new Gateway API.
+
+**2. How long have you been looking for a solution to previous problems?**
+
+Well, we deploy many projects so I would like to say: constantly. It's an iterative process. We always try to choose the best solutions for us and our customers, and the specific solutions change when a better one appears.
+
+**3. Were you comparing alternative solutions? Why us? Pros and cons? How long have you been using APISIX?**
+
+The fact that APISIX is under the umbrella of the Apache Software Foundation was a very big advantage.
+
+The rest came out of simple testing and proof-of-concept implementation.
+
+We are using and deploying APISIX for about one year.
+
+## About the Implementing Process
+
+**1. Would you share some details about how your team implemented APISIX?**
+
+All of our deployments of APISIX are within a Kubernetes cluster within AWS.
+
+We are installing it from the official helm charts, exposing it through AWS Network Load Balancer. We terminate the SSL at the NLB so we can take advantage of AWS Certificate Manager.
+
+We use the APISIX Ingress Controller to allow the users to configure the routes on their own, along with other manifests describing their Kubernetes application.
+
+We use the built-in Prometheus plugin to consume and process data metrics, to create alerting rules and dashboards visualizing the application state.
+
+**2. Were there any internal risks or additional costs involved with implementing APISIX? If so, how did you address them?**
+
+I don't think we took any risks. Granted, this was a new software, a new solution, but being an open-source one we could comfortably test it in a proof-of-concept scenario.
+
+It took some hours, but - I believe - not as much as would sending team members to workshops to learn an alternative, commercial solution.
+
+**3. How do you and your team currently use APISIX? What types of goals or tasks are you using APISIX to accomplish?**
+
+At the moment it is our go-to API Gateway of choice in Kubernetes environments.
+
+It allows us to easily set up an environment that can be perused by development teams in many projects to quickly prototype, create and deploy production-ready workloads.
+
+**4. What was the most obvious advantage you felt about APISIX?**
+
+Single most obvious advanteage: the ease of use.
+More advantages: being Kubernetes-native and open-source.
+
+## Achievements after Using Our Products
```

Comment:
```suggestion
## Achievements after Using APISIX
```

Reply:
Since the article is published on apisix.apache.org, let's do not use "our product" the term; instead, say APISIX directly